shotover-proxy/src/transforms/cassandra/sink_single.rs

```diff
@@ -123,13 +121,9 @@ impl CassandraSinkSingle {
                                         original,
                                         response: Err(err),
                                     } => {
-                                        responses.push(Message::new_response(
-                                            QueryResponse::empty_with_error(Some(
-                                                message::MessageValue::Strings(format!("{err}")),
-                                            )),
-                                            true,
-                                            original.original,
-                                        ));
+                                        let mut message = Message::from_frame(original.original);
```

Comment:
We can write this as 
![image](https://user-images.githubusercontent.com/5120858/153095864-c1c45d8d-751b-4a0f-96c8-2f80cdc74160.png)